src/content/news/2025-11-04-maplibre-newsletter-october-2025/index.mdx

```diff
@@ -0,0 +1,407 @@
+---
+title: "MapLibre Newsletter October 2025"
+date: "2025-11-04"
+categories: ["newsletter"]
+authors: ["ramya-ragupathy"]
+draft: false
+---
+
+import { Image } from "astro:assets";
+import radarLogo from "/public/img/radar-logo.svg";
+import maplibreLogo from "/public/img/maplibre-logos/maplibre-logo-for-light-bg.svg";
+import linedash from "./linedash.webp";
+import martin from "./martin.png";
+
+We begin this edition with sincere appreciation for [Radar](https://radar.com/), whose ongoing sponsorship helps sustain the MapLibre community.
+Welcome to the MapLibre Newsletter, October 2025 edition!
+
+<div
+  style={{
+    backgroundColor: "white",
+    padding: "2rem",
+    border: "10px solid #cfd8dc",
+    borderRadius: "0",
+    marginBottom: "3rem",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    gap: "2rem",
+    flexWrap: "nowrap",
+    overflow: "hidden",
+    flexDirection: "column",
+  }}
+>
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      gap: "2rem",
+      flexWrap: "nowrap",
+      width: "100%",
+    }}
+  >
+    <Image
+      src={maplibreLogo}
+      alt="MapLibre logo"
+      style={{
+        width: "40%",
+        maxWidth: "300px",
+        height: "auto",
+        objectFit: "contain",
+      }}
+    />
+    <Image
+      src={radarLogo}
+      alt="Radar logo"
+      style={{
+        width: "40%",
+        maxWidth: "330px",
+        height: "auto",
+        objectFit: "contain",
+      }}
+    />
+  </div>
+  <div
+    style={{
+      marginTop: "1rem",
+      fontSize: "1rem",
+      textAlign: "center",
+      color: "#263238",
+      fontStyle: "italic",
+    }}
+  >
+    Thank you Radar for your sustained support to MapLibre (2023–2025) 🙌
+  </div>
+</div>
+
+Before diving into the updates, we’ve added a new .github repository, giving the MapLibre GitHub organization a fresh, welcoming README! Check it out here: https://github.com/maplibre
+
+## 📱 MapLibre Native
+
+- Releases
+
+  <ul class="releases">
+    <li>
+      MapLibre Android:
+      [12.0.0](https://github.com/maplibre/maplibre-native/releases/tag/android-v12.0.0),
+      [12.0.1](https://github.com/maplibre/maplibre-native/releases/tag/android-v12.0.1)
+    </li>
+    <li>
+      MapLibre iOS:
+      [6.19.2](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.19.2),
+      [6.19.3](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.19.3)
+    </li>
+  </ul>
+
+- With the WebGPU support contributed by Birk Skyum in
+  [#3899](https://github.com/maplibre/maplibre-native/pull/3899), the WebGPU backend can now be used with either wgpu (new) or Dawn.
```

Comment:
Not an native speaker, but this reads a bit akward. lets split into shorter sentences


```suggestion
- Birk Skyum contributed WebGPU support in [#3899](https://github.com/maplibre/maplibre-native/pull/3899).
  This means now either wgpu (new) or Dawn can be used as a backend.
```